test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua

```diff
@@ -0,0 +1,198 @@
+-- and PolicyTableUpdate is failed by any reason even after retry strategy
+-- and "ForceProtectedService" is ON(0x07) at .ini file
+-- and app sends StartService (<any_serviceType>, encypted=true) to SDL
+-- SDL must respond StartService (NACK) to this mobile app
+--
+-- 1. Used preconditions:
+-- RPC SetAudioStreamingIndicator is allowed by policy
+-- ForceProtectedService is set to ON in .ini file
+-- Media app exists in LP, no certificate in module_config
+--
+-- 2. Performed steps
+-- 2.1. Register and activate application.
+-- 2.2. Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- 2.3. Send wrong policyfile (updated with function create_ptu_certificate_exist(false, true) )
+--
+-- Expected result:
+-- 1. Application is registered and activated successfully.
+-- 2. SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- SDL should not respond to StartService_request
+-- SDL should not process request to HMI
+-- 3. SDL must respond StartService (NACK) to this mobile app
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyCeritificates = require('user_modules/shared_testcases/testCasesForPolicyCeritificates')
+local events = require('events')
+local Event = events.Event
+
+--[[ General Precondition before ATF start ]]
+commonPreconditions:BackupFile("smartDeviceLink.ini")
+--commonFunctions:write_parameter_to_smart_device_link_ini("ForceProtectedService", "0x07")
+testCasesForPolicyCeritificates.update_preloaded_pt(config.application1.registerAppInterfaceParams.appID, false)
+testCasesForPolicyCeritificates.create_ptu_certificate_exist(false, true)
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+function Test:Precondition_First_StartService()
```

Comment:
@dboltovskyi, description is updated. Please see b45fd31ac007752d4ffc45809e9448e083e48387